pkgs/development/python-modules/latexrestricted/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchPypi,
+  buildPythonPackage,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "latexrestricted";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-AMfDTruJKejDdXW98VkeeEwELql5566bsL1SutLDpso=";
+  };
+
+  build-system = [
+    setuptools
+  ];
```

Comment:
Looks like that you need to run `nixfmt` again.

Reply:
I have run the following command, and it did not produced any further changes, and the automatic tests passed.

```console
nix-shell -p nixfmt-rfc-style --run "nixfmt pkgs/development/python-modules/latexrestricted/default.nix"
````

Reply:
Done.